packages/editor-ui/src/components/ParameterInputList.vue

```diff
@@ -191,18 +204,27 @@ export default mixins(workflowHelpers).extend({
 		node(): INodeUi | null {
 			return this.ndvStore.activeNode;
 		},
+		nodeAuthFields(): INodeProperties[] {
+			return getNodeAuthFields(this.nodeType);
+		},
 		indexToShowSlotAt(): number {
 			let index = 0;
+			// For nodes that use old credentials UI, keep credentials below authentication field in NDV
+			// otherwise credentials will use auth filed position since the auth field is moved to credentials modal
+			const fieldOffset = KEEP_AUTH_IN_NDV_FOR_NODES.includes(this.nodeType?.name || '') ? 1 : 0;
 			const credentialsDependencies = this.getCredentialsDependencies();
 
 			this.filteredParameters.forEach((prop, propIndex) => {
 				if (credentialsDependencies.has(prop.name)) {
-					index = propIndex + 1;
+					index = propIndex + fieldOffset;
 				}
 			});
 
 			return index < this.filteredParameters.length ? index : this.filteredParameters.length - 1;
 		},
+		mainNodeAuthField(): INodeProperties | null {
+			return getMainAuthField(this.nodeType || undefined);
```

Comment:
`|| undefined` looks odd. Maybe `getMainAuthField` should validate.

Reply:
Yeah, this should be updated